README.md

```diff
@@ -170,3 +170,13 @@ outputs:
 + stdout:
 +   enabled: true
 ```
+
+**cloudevents**
+
+The `cloudevents` driver emits [cloud events](https://cloudevents.io).
+
+```diff
+outputs:
++  cloudevents:
++    url: http://example.com:1234
```

Comment:
I might rename url to http, or have 
```
http:
  url:
```

Then you could support sending to a event broker (like pubsub or amqp) without needing to rework your config.